projects/core/src/checkout/facade/payment-type.service.ts

```diff
@@ -43,14 +43,19 @@ export class PaymentTypeService {
           select(getProcessStateFactory(GET_PAYMENT_TYPES_PROCESS_ID))
         )
       ),
-      tap(([, loadingState]) => {
+      tap(([_, loadingState]) => {
         if (
           !(loadingState.loading || loadingState.success || loadingState.error)
         ) {
           this.loadPaymentTypes();
         }
       }),
       pluck(0),
+      map((paymentTypes) => {
```

Comment:
need to remove this part to remove the sorting and have the list come out the same way as the backend